specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -472,71 +417,75 @@ namespace Microsoft.Windows.PushNotifications
     };
 
+    [feature(Feature_PushNotifications)]
     runtimeclass PushNotificationChannel
     {
-        PushNotificationChannel(String uri, Windows.Foundation.DateTime expiration);
-
         // The Channel Uri for app to Post a notification to.
-        String Uri { get; };
+        Windows.Foundation.Uri Uri { get; };
 
         // Expiration of the Channel
         Windows.Foundation.DateTime ExpirationTime { get; };
 
         // Unsubscribes the channel
         void Close();
-
-        // In-memory Event handler for Push Notifications
-        event Windows.Foundation.TypedEventHandler<PushNotificationChannel, PushNotificationReceivedEventArgs> PushReceived;
     }
 
+    [feature(Feature_PushNotifications)]
     runtimeclass PushNotificationCreateChannelResult
     {
         PushNotificationCreateChannelResult(
             PushNotificationChannel channel,
             HRESULT extendedError,
             PushNotificationChannelStatus status);
 
-        // The Push channel associated with the Result. Null if InProgress or completion failed
+        // The Push channel associated with the Result. Valid only if status is CompletedSuccess.
         PushNotificationChannel Channel { get; };
 
         // More detailed error code in addition to the ChannelStatus state.
         HRESULT ExtendedError{ get; };
 
-        // The Status of the ChannelComplete operation
+        // The Status of the CreateChannel operation
         PushNotificationChannelStatus Status { get; };
     };
 
-    static runtimeclass PushNotificationManager
+    [feature(Feature_PushNotifications)]
+    runtimeclass PushNotificationManager
     {
-        // Register an activator using an ActivationInfo context and cache the flag for unregister
-        static void RegisterActivator(PushNotificationActivationInfo details);
+        // Gets a Default instance of a PushNotificationManager
+        static PushNotificationManager Default{ get; };
+
+        // Registers an application for Notifications
+        // For Unpackaged applications, registers support for protocol activations
+        // For packaged applications with Background Infrastructure support, registers Push Trigger with the Background Infrastructure
+        // For packaged applications without Background Infrastructure support, registers a COM Service
+        void Register();
 
-        // Unregister any activator given PushNotificationRegistrationActivators
-        // 1) If kind = PushTrigger is specified, the trigger itself will be removed
-        // 2) If kind = ComActivator is specified, the Project Reunion Background Task component will no longer act as an InProc COM Server
-        // 3) If kind = ProtocolActivator is specified, the application will be unregistered from the long running process that handles activation
-        static void UnregisterActivator(PushNotificationRegistrationActivators activators);
+        // Packaged Apps: Force COM to launch a new process
```

Comment:
I'm sure you don't mean that calling `Unregister` will cause a new foo.exe to be launched just because of unregistration.  Maybe:

```cpp
// Stops the flow of notifications to this process and manager object. Additional incoming
// notifications may cause a new process to be launched to handle them. To stop recieving
// all notifications, use <this method name>.
```